ui/pages/notifications/index.scss

```diff
@@ -105,10 +105,6 @@
       flex-direction: column;
       margin-left: 12px;
 
-      &__message {
```

Comment:
Do we not need this anymore?

Reply:
Nope, our MarkDown renderer handles it :)